swap_meet/item.py

```diff
@@ -1,2 +1,33 @@
 class Item:
-    pass
+    def __init__(self, category=None, condition=0):
+        self.category = category if category is not None else ""
+        self.condition = condition
+    
+    def __str__(self):
+        return "Hello World!"
+    
+    def condition_description(self):
+        condition_num = int(self.condition)
+        conditions = {
+            0: "This should probably be in the free box.",
+            1: "You could say that it is well-loved?",
+            2: "Hand me down round seven.",
+            3: "There is potential here.",
+            4: "Honestly, what a score.",
+            5: "Still has the tags."
+        }
+        item_condition = conditions[condition_num]
+        return item_condition
```

Comment:
we can merge these two together:

```suggestion
        return conditions[condition_num]
```
or get rid of line 10 and:

```suggestion
        return conditions[int(self.condition)]
```